src/test/java/org/opentripplanner/transit/raptor/moduletests/B10_FlexAccess.java

```diff
@@ -1,7 +1,7 @@
 package org.opentripplanner.transit.raptor.moduletests;
```

Comment:
This test (`B10_FlexAccess`) and (`B11_FlexEgress`) doesn't have `*Test` in the name, and so by-default `mvn test` does not run them. They should probably be renamed to contain a `Test` suffix.

Reply:
Tank you for spotting this. I just renamed these.